addons/laser/functions/fnc_findStrongestRay.sqf

```diff
@@ -25,7 +39,7 @@ if(count _outliers > 0) then {
         _spot = [];
         _testPos = (_remainingSpots select 0);
         {
-            _samplePos = _x;
+            _x params ["_samplePos"];

```

Comment:
This would probably break the position check on line 43, as it's getting the first element from `_x`, which I guess to be a `position`/`array`.